modules/RTC/JitsiTrack.js

```diff
@@ -407,6 +407,16 @@ export default class JitsiTrack extends EventEmitter {
         return null;
     }
 
+    /**
+     * Returns the streaming status of the track.
+     *
+     * @returns {string} the streaming status <tt>TrackStreamingStatus</tt> of the track. Returns null if
+-    * trackStreamingStatusImpl hasn't been initialized.
+     */
+    getTrackStreamingStatus() {
```

Comment:
why is it needed on a local track? it's not supposed to be available for local tracks

Reply:
Sorry was too quick to merge this one. I found the bug in my JM PR, this change wasn't needed, will be reverting this one.